tools/script_extractor.py

```diff
@@ -230,9 +230,9 @@ def createList(): # this is a func just so all this can go at the bottom
 	("Func_d1b3", "", DO_NOT_QUIT),
 	("ScriptCommand_QuitScriptFully", "", QUIT_SPECIAL),
 	("Func_d244", "q", DO_NOT_QUIT),
-	("Func_d24c", "q", DO_NOT_QUIT),
+	("ScriptCommand_ShowMultichoiceTextbox_ChooseDeckToDuelAgainst", "q", DO_NOT_QUIT),
 	("ScriptCommand_OpenDeckMachine", "b", DO_NOT_QUIT),
-	("Func_d271", "q", DO_NOT_QUIT),
+	("ScriptCommand_ShowMultichoiceTextbox_ChooseStarterDeck", "q", DO_NOT_QUIT),
```

Comment:
This one would definitely be blank ("" instead of "q"). Again, I can fix these since I'm working on script extraction changes in my branch.